### PR TITLE
Feat/support modes llm whisperer

### PR DESCRIPTION
**Pull request by johnyrahul**
## What

- Updating SDK in services, and tools to use the latest mode option available at LLM whisperer adapter

## Why

-  LLM whisperer side has introduced new param  where it can access certain modes. Refer : 
https://github.com/Zipstack/unstract-sdk/pull/103
## How

-

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

-

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Review by chandrasekharan-zipstack — APPROVED** [review]

File: tools/classifier/requirements.txt
Comment (chandrasekharan-zipstack) [inline tools/classifier/requirements.txt:1]: @johnyrahul can you bump the tool versions also so that you can build them? Or will it be a separate PR?

**Review by chandrasekharan-zipstack — APPROVED** [review]

**Review by gaya3-zipstack — APPROVED** [review]
Looks fine.
